newhelm/main.py

```diff
@@ -59,6 +59,29 @@ def list_tests() -> None:
         click.echo(f"Description: {metadata.description}")
         click.echo()
 
+@newhelm_cli.command()
+def list_suts() -> None:
+    """List details about all registered SUTs."""
+    secrets = load_secrets_from_config()
+    for sut, sut_entry in SUTS.items():
+        try:
+            sut_obj = sut_entry.make_instance(secrets=secrets)
+        except MissingSecretValues as e:
+            display_header(
+                f"Cannot display SUT {sut} because it requires the following secrets:"
+            )
+            for secret in e.descriptions:
+                click.echo(secret)
+            click.echo()
+            continue
+        
+        display_header(sut)
+        click.echo(f"class={sut_obj.__class__.__name__}")
+        click.echo(f"args={sut_obj._initialization_record.args}") # Consider create getter function for `_initialization_record`
+        click.echo(f"kwargs={sut_obj._initialization_record.kwargs}")
+        click.echo(f"Uses required secrets:")
+        click.echo(secrets)
```

Comment:
`secrets` here is everything you defined in your `config/secrets.toml` file, and will be the same for all SUTs.

In order to get this behavior I think we are going to need to add functionality to [dependency_injection](https://github.com/mlcommons/newhelm/blob/bfba11f6383f9e7fdd0b114f4b896cc0be5df5e1/newhelm/dependency_injection.py#L12) that tells us what dependencies were used.